[libmount] Build error on x64-linux

Package: libmount:x64-linux -> 2.38#2

**Host Environment**

- Host: x64-linux
- Compiler: GNU 9.4.0
-    vcpkg-tool version: 2023-11-16-4c1df40a3c5c5e18de299a99e9accb03c2a82e1e
    vcpkg-scripts version: 45f6e57d3 2023-12-06 (2 hours ago)

**To Reproduce**

`vcpkg install sdl2`
**Failure logs**

```
-- Using cached util-linux-2.38.tar.xz.
-- Cleaning sources at /home/jule/lib/vcpkg/buildtrees/libmount/src/2.38-22af6c1200.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/jule/lib/vcpkg/downloads/util-linux-2.38.tar.xz
-- Using source at /home/jule/lib/vcpkg/buildtrees/libmount/src/2.38-22af6c1200.clean
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Generating configure for x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /usr/bin/autoreconf -vfi
    Working Directory: /home/jule/lib/vcpkg/buildtrees/libmount/src/2.38-22af6c1200.clean/
    Error code: 1
    See logs for more information:
      /home/jule/lib/vcpkg/buildtrees/libmount/autoconf-x64-linux-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:629 (vcpkg_execute_required_process)
  ports/libmount/portfile.cmake:19 (vcpkg_configure_make)
  scripts/ports.cmake:170 (include)



```
<details><summary>/home/jule/lib/vcpkg/buildtrees/libmount/autoconf-x64-linux-err.log</summary>

```
autoreconf: Entering directory `.'
autoreconf: running: autopoint --force
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 345.
autoreconf: failed to run autopoint: No such file or directory
autoreconf: autopoint is needed because this package uses Gettext
```
</details>

--- a/ports/libmount/portfile.cmake
+++ b/ports/libmount/portfile.cmake
@@ -12,26 +12,39 @@ vcpkg_extract_source_archive(
 
 set(ENV{GTKDOCIZE} true)
 
+vcpkg_list(SET options)
+if("nls" IN_LIST FEATURES)
+    vcpkg_list(APPEND options "--enable-nls")
+else()
+    set(ENV{AUTOPOINT} true) # true, the program
+    vcpkg_list(APPEND options "--disable-nls")
+endif()
+
 vcpkg_configure_make(
     AUTOCONFIG
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
+        ${options}
         --disable-asciidoc
         --disable-all-programs
+        --disable-dependency-tracking
         --enable-libmount
         --enable-libblkid
+        "--mandir=${CURRENT_PACKAGES_DIR}/share/man"
 )
 
 vcpkg_install_make()
 vcpkg_fixup_pkgconfig()
 
-if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")
-    file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/sbin" "${CURRENT_PACKAGES_DIR}/debug/sbin")
-endif()
 
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share")
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/tools") # empty folder
+file(REMOVE_RECURSE
+    "${CURRENT_PACKAGES_DIR}/debug/bin"
+    "${CURRENT_PACKAGES_DIR}/debug/sbin"
+    "${CURRENT_PACKAGES_DIR}/debug/share"
+    "${CURRENT_PACKAGES_DIR}/bin"
+    "${CURRENT_PACKAGES_DIR}/sbin"
+    "${CURRENT_PACKAGES_DIR}/share"
+    "${CURRENT_PACKAGES_DIR}/tools"
+)
 
-file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/README.licensing" "${SOURCE_PATH}/COPYING")

--- a/ports/libmount/portfile.cmake
+++ b/ports/libmount/portfile.cmake
@@ -1,11 +1,7 @@
-set(VERSION_MAJOR 2)
-set(VERSION_MINOR 38)
-set(VERSION ${VERSION_MAJOR}.${VERSION_MINOR})
-
 vcpkg_download_distfile(ARCHIVE
-    URLS "https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v${VERSION_MAJOR}.${VERSION_MINOR}/util-linux-${VERSION}.tar.xz"
+    URLS "https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v${VERSION}/util-linux-${VERSION}.tar.xz"
     FILENAME "util-linux-${VERSION}.tar.xz"
-    SHA512 d0f7888f457592067938e216695871ce6475a45d83a092cc3fd72b8cf8fca145ca5f3a99122f1744ef60b4f773055cf4e178dc6c59cd30837172aee0b5597e8c
+    SHA512 3d59a0f114c06be19ef7f86fca37ba5b9073823d011b3fc37997ddb00124b4505ea32903b78798a64dffbccf0ba645a692678ee845cc65a5b321824448a82a94
 )
 
 vcpkg_extract_source_archive(

--- a/ports/libmount/vcpkg.json
+++ b/ports/libmount/vcpkg.json
@@ -3,6 +3,21 @@
   "version": "2.39",
   "description": "Block device identification library from util-linux",
   "homepage": "https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/about/",
-  "license": "GPL-2.0-only",
-  "supports": "linux"
+  "license": null,
+  "supports": "linux",
+  "features": {
+    "nls": {
+      "description": "Enable native language support",
+      "dependencies": [
+        {
+          "name": "gettext",
+          "host": true,
+          "features": [
+            "tools"
+          ]
+        },
+        "gettext-libintl"
+      ]
+    }
+  }
 }

--- a/ports/libmount/vcpkg.json
+++ b/ports/libmount/vcpkg.json
@@ -1,7 +1,6 @@
 {
   "name": "libmount",
-  "version": "2.38",
-  "port-version": 2,
+  "version": "2.39",
   "description": "Block device identification library from util-linux",
   "homepage": "https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/about/",
   "license": "GPL-2.0-only",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4553,8 +4553,8 @@
       "port-version": 0
     },
     "libmount": {
-      "baseline": "2.38",
-      "port-version": 2
+      "baseline": "2.39",
+      "port-version": 0
     },
     "libmpeg2": {
       "baseline": "0.5.1",

--- a/versions/l-/libmount.json
+++ b/versions/l-/libmount.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5fbc2733330a3bac410e90c460db96f54e39e0a5",
+      "version": "2.39",
+      "port-version": 0
+    },
+    {
       "git-tree": "747604a5c97736ced9e0ef3b2963c2c949bdc655",
       "version": "2.38",
       "port-version": 2

